### PR TITLE
Update ray launcher document.

### PR DESCRIPTION
**Pull request by jieru-hu**
## Motivation

address #1121

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebookresearch/hydra/blob/master/CONTRIBUTING.md)?

Yes/No

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: website/docs/plugins/ray_launcher.md [hunk 1]
```diff
@@ -223,26 +223,25 @@ hydra.launcher.ray_cluster_cfg.provider.cache_stopped_nodes=true
...
+$ python my_app.py  --multirun hydra/launcher=ray_local +hydra.launcher.ray_init_cfg='{address: localhost:6379}'
+[2020-11-10 15:13:32,245][HYDRA] Ray Launcher is launching 1 jobs, sweep output dir: multirun/2020-11-10/15-13-32
+[2020-11-10 15:13:32,245][HYDRA] Initializing ray with config: {'num_cpus': None, 'num_gpus': None, 'address': 'localhost:6379'}
+2020-11-10 15:13:32,249 INFO worker.py:633 -- Connecting to existing Ray cluster at address: 10.30.99.17:6379
+[2020-11-10 15:13:32,274][HYDRA]        #0 : 
+(pid=93358) [2020-11-10 15:13:32,460][__main__][INFO] - Executing task 1
 ```
+
+### Configure `ray.init()` and `ray.remote()`
+Ray launcher is built on top of [`ray.init()`](https://docs.ray.io/en/master/package-ref.html?highlight=ray.remote#ray-init) and [`ray.remote()`](https://docs.ray.io/en/master/package-ref.html?highlight=ray.remote#ray-remote). You can configure `ray` by overriding `hydra.launcher.ray_init_cfg` and `hydra.launcher.ray_remote_cfg`. Check out [here](https://github.com/facebookresearch/hydra/blob/master/plugins/hydra_ray_launcher/examples/simple/config.yaml) for an example.
```
Comment (shagunsodhani) [inline website/docs/plugins/ray_launcher.md:247]:
```suggestion
Ray launcher is built on top of [`ray.init()`](https://docs.ray.io/en/master/package-ref.html?highlight=ray.remote#ray-init) and [`ray.remote()`](https://docs.ray.io/en/master/package-ref.html?highlight=ray.remote#ray-remote). You can configure `ray` by overriding `hydra.launcher.ray_init_cfg` and `hydra.launcher.ray_remote_cfg`. Check out an [example config](https://github.com/facebookresearch/hydra/blob/master/plugins/hydra_ray_launcher/examples/simple/config.yaml).
```

File: website/docs/plugins/ray_launcher.md [hunk 2]
```diff
@@ -223,26 +223,25 @@ hydra.launcher.ray_cluster_cfg.provider.cache_stopped_nodes=true
...
-[2020-11-02 16:46:58,267][HYDRA] Initializing ray with config: {'num_cpus': 1, 'num_gpus': 0}
-2020-11-02 16:46:58,827 INFO services.py:1164 -- View the Ray dashboard at http://127.0.0.1:8265
-[2020-11-02 16:46:59,888][HYDRA]        #0 : 
-(pid=85035) [2020-11-02 16:47:00,306][__main__][INFO] - Executing task 1
+$ python my_app.py  --multirun hydra/launcher=ray_local
+[2020-11-10 15:16:28,215][HYDRA] Ray Launcher is launching 1 jobs, sweep output dir: multirun/2020-11-10/15-16-28
+[2020-11-10 15:16:28,215][HYDRA] Initializing ray with config: {}
+2020-11-10 15:16:28,863 INFO services.py:1164 -- View the Ray dashboard at http://127.0.0.1:8266
+[2020-11-10 15:16:29,934][HYDRA]        #0 : 
+(pid=97801) [2020-11-10 15:16:30,453][__main__][INFO] - Executing task 1
```
Comment (omry) [inline website/docs/plugins/ray_launcher.md:233]: can you strip the timestamp from the output? this does not add value here.

File: plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/__init__.py [hunk 3]
```diff
@@ -19,9 +19,7 @@ class RayLocalLauncherConf(TargetConf):
         "hydra_plugins.hydra_ray_launcher.ray_local_launcher.RayLocalLauncher"
     )
     ray_init_cfg: Dict[str, Any] = field(default_factory=dict)
-    ray_remote_cfg: Dict[str, Any] = field(
-        default_factory=lambda: {"num_cpus": 1, "num_gpus": 0}
-    )
+    ray_remote_cfg: Dict[str, Any] = field(default_factory=dict)
```
Comment (omry) [inline plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/__init__.py:22]: Do not inherit from TargetConf, it is deprecated (I am surprised that you did not see a warning.

**Comment by jieru-hu** [comment]
> can you strip the timestamp from the output? this does not add value here.

Done. I always forget to do that.

>Do not inherit from TargetConf, it is deprecated (I am surprised that you did not see a warning.

Removed.

**Review by shagunsodhani — COMMENTED** [review]
lgtm

**Review by shagunsodhani — APPROVED** [review]

**Pull request merged**